src/spikeinterface/core/node_pipeline.py

```diff
@@ -471,7 +503,7 @@ def check_graph(nodes):
             assert parent in nodes, f"Node {node} has parent {parent} that was not passed in nodes"
             assert (
                 nodes.index(parent) < i
-            ), f"Node are ordered incorrectly: {node} before {parent} in the pipeline definition."
+            ), f"Node are ordered incorrectly: {node} before {parent} in the pipeline definition."
```

Comment:
```suggestion
            ), f"Nodes are ordered incorrectly: {node} before {parent} in the pipeline definition."
```

We can say The node is ordered incorrectly or Nodes are ordered incorrectly :)